### PR TITLE
Owner -> Manager

### PR DESCRIPTION
**Pull request by glibsm**

### CODE REVIEW

**Comment by breerly** [comment]
Hmmm... a bit hesitant to overload the term Controller, which has already experienced quite a bit of semantic diffusion...

**Comment by alsamylkin** [comment]
A Runner!
Or a Lifecycle.. <- my favorite.

File: service/service.go [hunk 1]
```diff
@@ -81,8 +81,8 @@ type serviceConfig struct {
 	Roles       []string `yaml:"roles"`
 }
 
-// newOwner creates a service owner from a set of module creation functions and options.
-func newOwner(modules []ModuleCreateFunc, options ...Option) (Owner, error) {
```
Comment (madhuravi) [inline service/service.go]: Change to newController

**Comment by madhuravi** [comment]
@breerly Controller was the name most of us were happy with. We do acknowledge it's a bit loaded but it captures this interface the best. Feel free to suggest alternatives if you think of any.

**Comment by breerly** [comment]
Glue/MVCS uses this term already to mean a different thing - if for perhaps just this reason it's probably worth thinking of something else, since we will be presenting the user with 2 concepts using the same name.

**Comment by breerly** [comment]
Of course, up to you guys.

**Comment by glibsm** [comment]
Correct, that's our worry as well.

rails/expressjs background was always my concern. Most frameworks use `controller` to mean `handler`.

**Comment by glibsm** [comment]
However, `Owner` is also not ideal :)

**Comment by dmcaulay** [comment]
`Lifecycle`?

**Comment by glibsm** [comment]
`type ThisTooReallyDifficultToNameSorry interface`

**Comment by anuptalwalkar** [comment]
It's not a `Lifecycle` We need a verb. `service.Manager` is nice.

**Comment by madhuravi** [comment]
@breerly good point about the conflict with glue. We want to avoid that.
We decided to rename to Manager.

File: service/host.go [hunk 2]
```diff
@@ -57,9 +57,9 @@ type host struct {
 	started        bool
 }
 
-// TODO(glib): host is both an Owner and a Host?
+// TODO(glib): host is both an Manager and a Host?
 var _ Host = &host{}
-var _ Owner = &host{}
+var _ Manager = &host{}
```
Comment (madhuravi) [inline service/host.go]: Rename private struct to manager too.

**Comment by alsamylkin** [comment]
service.Interface!

**Comment by sectioneight** [comment]
service.Thing
service.WTF
service.TODO
service.ShedForBikes
service.Uber
service.Bag

**Pull request merged**